src/quo2/core.cljs

```diff
@@ -62,20 +65,19 @@
     quo2.components.settings.privacy-option
     quo2.components.share.qr-code.view
     quo2.components.share.share-qr-code.view
-    quo2.components.onboarding.small-option-card.view
-    quo2.components.tabs.segmented-tab
     quo2.components.tabs.account-selector
+    quo2.components.tabs.segmented-tab
     quo2.components.tabs.tabs
     quo2.components.tags.context-tags
-    quo2.components.tags.status-tags
     quo2.components.tags.permission-tag
+    quo2.components.tags.status-tags
     quo2.components.tags.tag
     quo2.components.tags.tags
     quo2.components.tags.token-tag
-    quo2.components.text-combinations.title.view
-    quo2.components.list-items.user-list))
+    quo2.components.text-combinations.title.view))
 
-(def toast quo2.components.notifications.toast/toast)
+(def notification quo2.components.notifications.notification.view/notification)
```

Comment:
can you put these in a commented section below like the rest. keeping it alpha sorted would be great too 👍 

i.e 
```
;;;; Notifications
(def notification quo2.components.notifications.notification.view/notification)
(def toast quo2.components.notifications.toast.view/toast)

```